### DISCUSSION TITLE
wiki/yuukapanel/index

### DISCUSSION THREAD

**Discussion by giscus** [YuukaPanelNew/YuukaPanelNew.github.io#2] [General]
# wiki/yuukapanel/index

幽香面板 四周年诚意之作幽香面板是一个小巧精致的开服面板 前置环境：NET8 功能介绍：  性能监控：在面板界面左上角常驻显示CPU和内存的使用占比  多服务器管理：一个面板可同时管理多个服务器，或管理多个不同游戏的服务器（目前支持原生BDS、BDSLL、JE、幻兽帕鲁等）  快捷操作：继承于幽香面板2，服务器配置页面可快捷发送指令，添加或移除白名单、管理员，踢出玩家等  性能限额：可自由限制或预

https://yuukapanelnew.github.io/wiki/yuukapanel/index.html

**Comment by ExquisiteCore** [comment]
测试wiki评论系统
